src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -37,11 +56,10 @@ public override void WillRemoveSubview(UIView uiview)
 				return null;
 			}
 
-			if (!result.UserInteractionEnabled)
+			if (Descendants is not null && Descendants.Contains(result) && !result.UserInteractionEnabled)
```

Comment:
Here, the HitTest result could be an element of the hierarchy, but we are really only interested in elements of the hierarchy that come from Handlers, that is, those native elements created from an abstraction and that inherit from View and can use the InputTransparent property.

In the case of the WebView, the first element was `_UIRemoteView` with UserInteractionEnabled equals to false. This blocked interaction with the WebView. With the changes, we only verify elements added as children of the Layout, in the example added in this PR, it would only be a single `WKWebView` element, our WebView that is not disabled nor does use InputTransparent.